doc/radosgw/placement.rst

```diff
@@ -239,8 +239,8 @@ name in an HTTP header with the request. The S3 protocol uses the
 ``X-Amz-Storage-Class`` header, while the Swift protocol uses the
 ``X-Object-Storage-Class`` header.
 
-When using AWS S3 SDKs such as python boto3, it is important that the non-default
-storage class will be called as one on of the AWS S3 allowed storage classes, or else the SDK
+When using AWS S3 SDKs such as ``boto3``, it is important that non-default
+storage class names are among those provided by AWS S3, or else the SDK
```

Comment:
`storage class names match those used by AWS S3`
and include a list of those names?

Reply:
I struggled for a while with the right phrasing; I like yours better.  My goal was to clarify the wording; I'd read the original sentence several times before getting the point.

I didn't want to include a list of those names, because I imagine they will change over time, resulting in divergence. I also thought of including a link to AWS, but external URLs can also change over time.  @zdover23  what do you think?